### PR TITLE
feat: add missing fields

### PR DESCRIPTION
**Pull request by cazala**
This PR adds the fields `creator` and `searchIsStoreMinter` to the `Item`, and `searchIsStoreMinter` to the `Collection`.

This has already been deployed to the Mumbai subgraph: https://thegraph.com/legacy-explorer/subgraph/decentraland/collections-matic-mumbai

### CODE REVIEW

File: src/handlers/collection.ts [hunk 1]
```diff
@@ -230,6 +264,32 @@ export function handleSetGlobalMinter(event: SetGlobalMinter): void {
...
+          // check if store is item minter
+          let isStoreItemMinter = false
+          let itemMinters = item.minters
+          for (let j = 0; j < item.minters.length; j++) {
+            if (storeAddress == itemMinters![i]) {
+              isStoreItemMinter = true
+            }
+          }
+          // if not, unset flag on item
+          if (!isStoreItemMinter) {
```
Comment (nachomazzara) [inline src/handlers/collection.ts:285]: I guess that we can just use `item.searchIsStoreMinter = isStoreItemMinter`. WDYT?

File: schema.graphql [hunk 2]
```diff
@@ -15,12 +15,16 @@ type Collection @entity {
   createdAt: BigInt!
   updatedAt: BigInt!
   reviewedAt: BigInt!
+
+  # Searcheable fields
+  searchIsStoreMinter: Boolean
```
Comment (nachomazzara) [inline schema.graphql:20]: I think that we can add `Boolean!` here to force a value.

**Review by nachomazzara — CHANGES_REQUESTED** [review]
Excellent!

File: src/handlers/collection.ts (hunk 1 above)
Comment (cazala) [reply src/handlers/collection.ts:285]: Do you mean removing the if? Bc that would not be the same. I want to set it to `false` only when `isStoreItemMinter` is false, but I don't want to set it `true` when `isStoreItemMinter` is true.

File: schema.graphql [hunk 3]
```diff
@@ -44,6 +48,7 @@ type Item @entity {
 
   ## Collection fields
   searchIsCollectionApproved: Boolean
+  searchIsStoreMinter: Boolean
```
[the remark this answers is not in the document]
Comment (cazala) [reply schema.graphql]: done ✅

File: src/handlers/collection.ts [hunk 4]
```diff
@@ -116,11 +123,18 @@ export function handleAddItem(event: AddItem): void {
...
+    collectionContract.getChainId().toString() +
+    '/' +
+    collectionAddress +
+    '/' +
+    itemId
   item.urn = getURNForWearableV2(collectionAddress, itemId.toString())
   item.image = getItemImage(item)
   item.minters = []
   item.managers = []
+  item.searchIsStoreMinter = false
```
[the remark this answers is not in the document]
Comment (cazala) [reply src/handlers/collection.ts:138]: done ✅

File: schema.graphql (hunk 2 above)
Comment (cazala) [reply schema.graphql]: done ✅

File: src/handlers/collection.ts (hunk 1 above)
Comment (cazala) [reply src/handlers/collection.ts:286]: AH wait you are right, this is the item's minter, not the collection's minter. Brain fart.

**Review by nachomazzara — APPROVED** [review]

**Pull request merged**